app/adminPage.html

```diff
@@ -126,6 +126,20 @@ <h4 class="search-heading">Filter Results</h4>
                         <canvas id="myChart" width="100" height="100"></canvas>
                     </div>
                 </div>
+                <section class="categorySection">
+                    <form class="categoriesForm">
+                        <h2>Add New Category</h2>
+                        <label for="categoryName">Category Name :</label>
+                        <input name="categoryName" type="text" id="categoryName">
+                        <input id="addCategory" type="submit" class="btn primary-button submit" value="Add new category">
+                    </form>
```

Comment:
<div class="message-target">
<div id="categorySubmissionConfirmation" class="alert" role="alert"></div>
    </div>

Can we add this please so we have space to confirm successful addition?